plugin/latexlivepreview.vim

```diff
@@ -207,6 +207,10 @@ EEOOFF
 
     lcd -
 
+    if s:previewer == 'mupdf'
+        let b:livepreview_buf_data['run_cmd'] .= ' && kill -s HUP $(pgrep mupdf)'
+    endif
```

Comment:
Can you add some code to check the availability of the `pgrep` command and some comments to explain these three lines? Note that `pgrep` is not a standard program and many POSIX OSes do not have it. We can merge after these are done. Thanks for your contribution.